src/views/Incorporation/IncorporationAgreement.vue

```diff
@@ -79,18 +79,20 @@
                 to help you prepare your Incorporation Agreement and Articles.
               </div>
             </div>
-            <div class="help-div">
-              <h3>Retain the signed Incorporation Agreement and {{ entityDescription }} Articles</h3>
-              <ul>
-                <li>
-                  The company is required to keep signed copies of the Incorporation Agreement and
-                  Articles in the company’s record book. For a complete list of records a company
-                  is required to keep please see <a :href="section42Url" target="_blank">section 42</a>
-                  of the Business Corporations Act.
-                </li>
-              </ul>
-            </div>
           </template>
+
+          <div class="help-div">
+            <h3>Retain the signed Incorporation Agreement and {{ entityDescription }} Articles</h3>
+            <ul>
+              <li>
+                The company is required to keep signed copies of the Incorporation Agreement and
+                Articles in the company’s record book. For a complete list of records, a company
```

Comment:
Again, this is not the usual apostrophe that we've been using. I'm OK with changing it but we should try to be consistent.

Reply:
I can change all if it's for consistency.

Reply:
I prefer the simple apostrophe (the key beside Enter) instead of some fancy high-ASCII character :)

At least, that's what we've used elsewhere.

Reply:
Updated with correct one for consistency